experimental/sql/dialects/ibis_dialect.py

```diff
@@ -276,34 +276,29 @@ def get(table: Region, metrics: Region, names: list[str]) -> 'Aggregation':
 
 
 @irdl_op_definition
-class InnerJoin(Operation):
+class CartesianProduct(Operation):
   """
-  Models an ibis inner join of table `left` with table `right` and the condition
-  `predicates`. An empty region for `predicates` means that this operation is
-  just a Cartesian product.
+  Models an ibis Cartesian product of tables `left` and `right`.
```

Comment:
Maybe it makes sense to add a short definition à la ", i.e., the combination of all elements from `left` and `right`"?